File: app/src/main/java/com/example/water/cproject/Accelerometer.java
package com.example.water.cproject;

/**
 * Created by water on 2017-04-18.
 */

@SuppressWarnings("DefaultFileTemplate")
public class Accelerometer {
    private float x, y, z;

    public Accelerometer() {
        updateData(0,0,0);
    }

    public void updateData(float x, float y, float z) {
        this.x = x;
        this.y = y;
        this.z = z;
    }

    public Accelerometer getData() {
        return this;
    }
    public float getX() {
        return x;
    }
    public float getY() {
        return y;
    }
    public float getZ() {
        return z;
    }
}

File: app/src/main/java/com/example/water/cproject/Genuino/Accelerometer.kt
package com.example.water.cproject.Genuino

/**
 * Created by water on 2017-04-18.
 */

class Accelerometer {
    var x: Float = 0.toFloat()
        private set
    var y: Float = 0.toFloat()
        private set
    var z: Float = 0.toFloat()
        private set

    val data: Accelerometer
        get() = this

    init {
        updateData(0f, 0f, 0f)
    }

    fun updateData(x: Float, y: Float, z: Float) {
        this.x = x
        this.y = y
        this.z = z
    }
}
